### ISSUE TITLE
Installation notes

### ISSUE THREAD

**Issue by eteq** [astropy/astrowidgets#18]
Somewhere (README? Docs?) there should be a bit more on the installation.  Don't have time to take care of it right now, but some bullet points I'm reminding myself to include because they tripped me up:

For jupyterlab:
* Needs an up-to-date jupyterlab
* Needs a reasonably up-to-date nodejs
* Need to have `ipywidgets` and `ipyevents` *in the kernel of execution*
* Need to do ``jupyter labextension install @jupyter-widgets/jupyterlab-manager`` *in the kernel of jupyterlab*
* Need to do ``jupyter labextension install @jupyter-widgets/jupyterlab-manager ipyevents`` (possible duplicate of the above, but I had to do them in order to get it to build properly...)

(I still haven't gotten ipyevents working though, so might have more to add)

**Comment by eteq** [comment]
Hooray, it works!  The key missing piece: `ipyevents` must be <=0.3.1

**Comment by pllim** [comment]
I prefer it to be in the docs. I added RTD badge to README, so should be discoverable? However, adding stuff to doc should wait till after #7.

**Comment by pllim** [comment]
I am working on this. Hold your horses.
